script.js

```diff
@@ -3,19 +3,50 @@ document.getElementById('add-task-btn').addEventListener('click', function() {
     const taskList = document.getElementById('task-list');
     if (taskInput.value.trim() !== "") {
       const newTask =document.createElement('li');
-      newTask.innerHTML = `
-      <span>${taskInput.value}</span>
-      <button class="editbtn">Edit</button>
-      <button class="deletebtn">Delete</button>
-    `;
-      deletebtn(newTask); 
-      editbtn(newTask); 
+      newTask.classList.add('task-item');
+      const checkbox = document.createElement('input');
+       checkbox.type = 'checkbox';
+        checkbox.classList.add('task-checkbox');
+         const taskText = document.createElement('span');
+          taskText.textContent = taskInput.value;
+    const editButton = document.createElement('button');
+    editButton.classList.add('editbtn');
+    editButton.textContent = 'Edit';
```

Comment:
Here also indentation is unsuitable.

Corrected - 
```
      newTask.classList.add('task-item');
      const checkbox = document.createElement('input');
      checkbox.type = 'checkbox';
      checkbox.classList.add('task-checkbox');
      const taskText = document.createElement('span');
      taskText.textContent = taskInput.value;
      const editButton = document.createElement('button');
      editButton.classList.add('editbtn');
      editButton.textContent = 'Edit';
```

i.e make it align to 
```
      const newTask =document.createElement('li');
```
indentation here.

Note - Make the indentation proper wherever you find.